src/test/groovy/graphql/introspection/IntrospectionTest.groovy

```diff
@@ -421,7 +421,7 @@ class IntrospectionTest extends Specification {
                                                         .name("inputField")
                                                         .type(GraphQLString))
                                                 .build())
-                                        .defaultValue(new FooBar())
+                                        .defaultValue(new FooBar()) // Retain for test coverage. There is no alternative method that sets an internal value.
```

Comment:
The alternatives are `defaultValueLiteral(Value)` or `defaultValueProgrammatic(Object)`, neither of which work in this case.